tests/attr/neuron/test_neuron_ablation.py

```diff
@@ -214,7 +214,7 @@ def _ablation_test_assert(
         baselines=None,
         neuron_index=0,
         attribute_to_neuron_input=False,
-    ):
+    ) -> None:
```

Comment:
Can you also add type annotations to parameters of this method, _ablation_test_assert?